ibu-imager/seedcreator/seedcreator.go

```diff
@@ -28,6 +29,7 @@ const (
 	// Default location for etcdStaticPodFile
 	etcdStaticPodFile      = "/etc/kubernetes/manifests/etcd-pod.yaml"
 	etcdStaticPodContainer = "etcd"
+	installationConfigurationFilesDir = "/usr/local/installation_configuration_files"
```

Comment:
The go formatter is causing the CI to fail here because of the whitespace misalignment